pkg/parser/parser.go

```diff
@@ -13,7 +13,7 @@ import (
 var (
 	//go:embed cse_cmd.sh.gtpl
 	bootstrapTrigger         string
-	bootstrapTriggerTemplate = template.Must(template.New("triggerBootstrapScript").Funcs(getFuncMap()).Parse(bootstrapTrigger)) //nolint:gochecknoglobals
+	bootstrapTriggerTemplate = template.Must(template.New("triggerBootstrapScript").Parse(bootstrapTrigger)) //nolint:gochecknoglobals
```

Comment:
for my own understanding since im not too clear on how the embedding works, but why can we get rid of the `.Funcs(getFuncMap())`?

Reply:
The reason I removed the getFuncMap was because it was using 2 functions depending on FeatureState, which I removed in this PR. Now I update the codes to keep the getFuncMap function returning an empty FuncMap.
Btw, this parser.go will be used for our upcoming parser which we haven't started the work yet. We will have a major development on parser soon. So anything in the parser.go or utilities.go are just temporary.

In case you need more context about go template, let me elaborate as follows.
Here we call `Funcs(getFuncMap())` so that the template `bootstrapTriggerTemplate` (which imports `cse_cmd.sh.gtpl`) can call some additional functions during parsing.
For example, if I add a `funcA` to the function map, then in the `cse_cmd.sh.gtpl` we can use it like `ENV_A= {{funcA .varA}}` where `funcA` takes an argument `.varA`. Hope this helps. 😊